.github/workflows/cd.yml

```
@@ -0,0 +1,45 @@
+name: CD (JIB 이미지 & 무중단 배포)
+
+on:
+  push:
+    branches: [ "develop", "feat/#5/cicd" ]
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.9)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)